conmon/conmon.c

```diff
@@ -975,7 +980,6 @@ int main(int argc, char *argv[])
 		exit(0);
 	}
 
-	// why not nexit?
 	if (opt_cid == NULL) {
 		fprintf(stderr, "Container ID not provided. Use --cid\n");
```

Comment:
Not related to this PR, but you could probably use `nexit` here instead to match the others below it.

Reply:
It turns out nexit uses opt_cid to print ctrID with the error; if it isn't provided it would segfault

Reply:
oh okay, got it! thanks for the info :)